share/picongpu/examples/IncidentField/include/picongpu/param/incidentField.param

```diff
@@ -33,42 +33,64 @@ namespace picongpu
     {
         namespace incidentField
         {
+            // This base class is merely to keep all laser-related things together
             class FunctorBase
             {
             public:
                 // Constants for pulse generation
-                static constexpr float_64 TIME_PERIOD_SI = 20 * SI::DELTA_T_SI;
-                static constexpr float_64 WAVE_LENGTH_SI = 2.0 * PI * TIME_PERIOD_SI * SI::SPEED_OF_LIGHT_SI;
-                static constexpr float_64 UNITCONV_A0_to_Amplitude_SI = -2.0 * PI / WAVE_LENGTH_SI
-                    * SI::ELECTRON_MASS_SI * SI::SPEED_OF_LIGHT_SI * SI::SPEED_OF_LIGHT_SI / SI::ELECTRON_CHARGE_SI;
+                static constexpr float_64 TIME_PERIOD_SI = 0.038e-9; // 0.038 ns
+                static constexpr float_64 WAVE_LENGTH_SI = TIME_PERIOD_SI * SI::SPEED_OF_LIGHT_SI;
+                static constexpr float_64 UNITCONV_A0_to_Amplitude_SI = -1.0 / WAVE_LENGTH_SI * SI::ELECTRON_MASS_SI
+                    * SI::SPEED_OF_LIGHT_SI * SI::SPEED_OF_LIGHT_SI / SI::ELECTRON_CHARGE_SI;
                 static constexpr float_64 A0 = 8.0;
                 static constexpr float_64 AMPLITUDE_SI = A0 * UNITCONV_A0_to_Amplitude_SI;
-                static constexpr uint32_t DURATION_STEPS = 300;
+                static constexpr float_64 DURATION_SI = 0.6e-9; // 0.6 ns
+
                 /// These need to be changed together with grid size
-                static constexpr uint32_t CENTER_X_CELLS = 80;
-                static constexpr uint32_t WIDTH_X_CELLS = 40;
-                static constexpr uint32_t CENTER_Y_CELLS = 40;
-                static constexpr uint32_t WIDTH_Y_CELLS = 20;
+                static constexpr float_64 CENTER_X_SI = 80.0e-3; // 80 mm

```

Comment:
With a grid of 128 x 128 x 128, as written in the `1.cfg` and `4.cfg`, neither `CENTER_X` nor `CENTER_Y`will be in the middle of the box. The middle of the box would be 64.0e-3 with a cell size of 1.e-3 as defined in `grid.param`.

Reply:
Those were center of the laser, not box

Reply:
Maybe it should be `LASER_CENTER_X_SI` to clarify that it's the center of the laser, not of the simulation box